package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-native-msal",
+  "name": "@nbt-private/react-native-msal",
```

Comment:
How would I get access to this private registry locally? I guess I can follow the instructions in the pipelines portal. But then what about users who do not have access to pipelines?

To me this raises a question, do we even need this to be an actual private registry? Can't we just publish to npm?

Reply:
cc @Marvedog

Reply:
that yes might be an issue. workaround would be to have person with access generate api key for pulling - tho this can be a blocker. private registries are behind a paywall in npm but we could switch. if private registry is not important then we can just have a public one just as well.

used what was available to us for free.

Reply:
Preferably, all users involved in the development should have access to Azure Pipelines and consequently access to this private registry so I don't see this as a major blocker.

On the other hand. I have no issue publishing to npm.

Reply:
For this npm org would be required - then we could have public package under for example `@nbt` scope - think this would be fine. @Marvedog let me know what you think

Reply:
That sounds fine. We can create an npm organization that we can use for this tommorow.